NotesApp/Scenes/List/ListViewController.swift

```diff
@@ -66,20 +73,24 @@ final class ListViewController: UIViewController {
             subview.translatesAutoresizingMaskIntoConstraints = false
         }
         
+        setupHierarchy()
+    }
+    
+    private func setupHierarchy() {
```

Comment:
Would be better to name this function setupConstraints or something similar as you are only setting constrains here